aws/setStackContext.sh

```diff
@@ -142,21 +142,42 @@ esac
 STACK_NAME="${STACK_NAME:-${PRODUCT_PREFIX}${SEGMENT_SUFFIX}${LEVEL_SUFFIX}${DEPLOYMENT_UNIT_SUFFIX}${DEPLOYMENT_UNIT_SUBSET_SUFFIX}}"
 TEMPLATE="${LEVEL_PREFIX}${DEPLOYMENT_UNIT_PREFIX}${DEPLOYMENT_UNIT_SUBSET_PREFIX}${REGION_PREFIX}template.json"
 STACK="${LEVEL_PREFIX}${DEPLOYMENT_UNIT_PREFIX}${DEPLOYMENT_UNIT_SUBSET_PREFIX}${REGION_PREFIX}stack.json"
+CHANGE="${LEVEL_PREFIX}${DEPLOYMENT_UNIT_PREFIX}${DEPLOYMENT_UNIT_SUBSET_PREFIX}${REGION_PREFIX}change.json"
```

Comment:
Do we need dryrun? Isn't change the same? So we always create the change set, but don't action it if DRYRUN is true...

Reply:
The idea I had was that CHANGE contains the changes that were made as part of the last run. DRYRUN shows the changes that would occur on the next run. Given that the DRYRUN changes are show echp'd maybe we don't need this?

Reply:
Removed the DRYRUN file output. Instead the Changes will just be displayed